src/java/org/apache/cassandra/security/FileBasedSslContextFactory.java

```diff
@@ -102,15 +116,20 @@ private boolean hasTruststore()
     public synchronized void initHotReloading()
     {
         boolean hasKeystore = hasKeystore();
+        boolean hasOutboundKeystore = hasKeystore();
```

Comment:
I think you want to call `hasOutboundKeystore()` here. This is just an example that code duplication makes it error prone.